src/app.js

```diff
@@ -44,9 +45,38 @@ class Deezer {
         });
         
         this.window = new Window(this);
+        this.registerDBusTheme(this.window);
         this.registerMediaKeys(this.window);
         initIPC(this.window);
     }
+
+    registerDBusTheme(window) {
+        let sessionBus = dbus.sessionBus();
+        // custom implementation of freedesktop darkmode switching
+        // because electron does not fetch this yet
+        sessionBus.getService('org.freedesktop.portal.Desktop').getInterface(
+            '/org/freedesktop/portal/desktop',
+            'org.freedesktop.portal.Settings', function(err, settings) {
+        
+                settings.Read('org.freedesktop.appearance', 'color-scheme', function(err, resp) {
```

Comment:
Read shouldn't be used anymore, use ReadOne instead.
https://github.com/flatpak/xdg-desktop-portal/pull/877